user-service/src/main/resources/application.yaml

```diff
@@ -38,15 +37,6 @@ management:
     shutdown:
       enabled: true
 
-feign:
-  client:
-    config:
-      default:
-        connectTimeout: 5000
-        readTimeout: 5000
-        loggerLevel: basic
-    default-config: default
-
```

Comment:
This section still exists in most of the other service application.yaml files. Are you sure it should be removed here and not in the other application.yaml files?

Reply:
readded